systemd/systemd.go

```diff
@@ -357,7 +357,6 @@ X-Snappy=yes
 [Service]
 ExecStart=/usr/bin/ubuntu-core-launcher {{.UdevAppName}} {{.AaProfile}} {{.FullPathStart}}
 Restart={{.Restart}}
-WorkingDirectory=/var/lib{{.SnapPath}}
```

Comment:
Should we keep that one?  This is not in disagreement with the core idea that the application is in CWD when launched. It's just that systemd itself is setting the CWD before calling out the app in this case.